src/exporters/era5_land.py

```diff
@@ -0,0 +1,178 @@
+    'evapotranspiration', 'forecast_albedo', 'lake_bottom_temperature',
+    'lake_ice_depth', 'lake_ice_temperature', 'lake_mix_layer_depth',
+    'lake_mix_layer_temperature', 'lake_shape_factor',
+    'lake_total_layer_temperature', 'leaf_area_index_high_vegetation',
+    'leaf_area_index_low_vegetation', 'potential_evaporation', 'runoff',
+    'skin_reservoir_content', 'skin_temperature', 'snow_albedo', 'snow_cover',
+    'snow_density', 'snow_depth', 'snow_depth_water_equivalent',
+    'snow_evaporation', 'snowfall', 'snowmelt', 'soil_temperature_level_1',
+    'soil_temperature_level_2', 'soil_temperature_level_3',
+    'soil_temperature_level_4', 'sub_surface_runoff',
+    'surface_latent_heat_flux', 'surface_net_solar_radiation',
+    'surface_net_thermal_radiation', 'surface_pressure', 'surface_runoff',
+    'surface_sensible_heat_flux', 'surface_solar_radiation_downwards',
+    'surface_thermal_radiation_downwards', 'temperature_of_snow_layer',
+    'total_precipitation', 'volumetric_soil_water_layer_1',
+    'volumetric_soil_water_layer_2', 'volumetric_soil_water_layer_3',
+    'volumetric_soil_water_layer_4'
+]
+
+
+class ERA5LandExporter(CDSExporter):
+    dataset = 'reanalysis-era5-land'
+    granularity = 'hourly'
+
+    @staticmethod
+    def print_valid_vars():
+        print(VALID_ERA5_LAND_VARS)
+
+    def create_selection_request(self,
+                                 variable: str,
+                                 selection_request: Optional[Dict] = None,
+                                 granularity: str = 'hourly',) -> Dict:
+        # setup the default selection request
+        assert variable in VALID_ERA5_LAND_VARS, "Need to select a variable " \
+            f"from {VALID_ERA5_LAND_VARS}"
+
+        processed_selection_request = {
+            'format': 'netcdf',
+            'variable': [variable]
+        }
+        for key, val in self.get_default_era5_times(granularity, land=True).items():
+            processed_selection_request[key] = val
+
+        # by default, we investigate Kenya
+        kenya_region = get_kenya()
+        processed_selection_request['area'] = self.create_area(kenya_region)
+
+        # update with user arguments
+        if selection_request is not None:
+            for key, val in selection_request.items():
+                if key in {'day', 'hour'}:
+                    warnings.warn(f'Overriding default {key} values. '
+                                  f'The ERA5LandExporter assumes all days '
+                                  f'and times (the default) are being downloaded')
+                val = self._check_iterable(val, key)
+                processed_selection_request[key] = [self._correct_input(x, key) for x in val]
+        return processed_selection_request
+
+    def export(self,
+               variable: str,
+               show_api_request: bool = True,
+               selection_request: Optional[Dict] = None,
+               break_up: Optional[str] = 'yearly',
```

Comment:
Based on the previous experiments, do we want to break things up?

Reply:
annually seems to work really well!

Reply:
(hacve exported quite alot of this)